projects/arlas-toolkit/src/lib/tools/utils.ts

```diff
@@ -78,6 +78,11 @@ export interface CollectionUnit {
   ignored: boolean;
 }
 
+export function getCollectionUnit(units: Array<CollectionUnit>, collection: string): string {
+  return units.find(u => u.collection === collection) ?
```

Comment:
I would store the units.find in a variable in order to avoid recalculating it twice